notify/main.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+import logging
+import os
+import socket
+import sys
+
+import flask
+import yaml
+
+# from salesforce import OAuth2, Client, send_to_sfdc
+import salesforce
+
+app = flask.Flask(__name__)
+
+
+def pretty_js(x):
+    return json.dumps(x, sort_keys=True, indent=4)
+
+
+def load_config(config_file):
+    with open(config_file) as fp:
+        config = yaml.safe_load(fp)
+    return config
+
+
+@app.errorhandler(404)
+def not_found(error):
+    return flask.jsonify({"error": "Not Found"}), 404
+
+
+@app.route('/api/v1/nagios_data', methods=['POST'])
+def nagios_data():
```

Comment:
This API requires some love: 

- It should be nagios independent 
- API method should be called "/api/v1/alert" 
- Python method should be called def send_alert()
- It should validate POST content before doing any operations with it